src/main/java/ch/njol/skript/effects/EffGlowingText.java

```diff
@@ -0,0 +1,96 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Sign;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.BlockStateMeta;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Make Sign Glow")
+@Description("Makes a sign (either a block or item) have glowing text or normal text")
+@Examples({"make target block of player have glowing text"})
+@Since("INSERT VERSION")
+public class EffGlowingText extends Effect {
+
+	static {
+		if (Skript.methodExists(Sign.class, "setGlowing", boolean.class))
+			Skript.registerEffect(EffGlowingText.class,
+				"make %blocks/itemtypes% have glowing text",
+				"make %blocks/itemtypes% have (normal|non[-| ]glowing) text");
```

Comment:
```suggestion
		if (Skript.methodExists(Sign.class, "setGlowing", boolean.class)) {
			Skript.registerEffect(EffGlowingText.class,
				"make %blocks/itemtypes% have glowing text",
				"make %blocks/itemtypes% have (normal|non[-| ]glowing) text"
			);
		}	
```
I recommend braces here since it goes multiple lines :)